docs/faq.md

```diff
@@ -41,18 +42,20 @@ The Chrysalis release still depends on the Coordinator/Compass to run and issue
 
 ## What are Spent Addresses and Why are they Dangerous?
+
 - In the IOTA 1.0 network, IOTA used Winternitz One Time Signatures (W-OTS) - think of these as, more or less, an authenticator and validator for a transaction. These keys and signatures are highly secure against malicious attacks for signing transactions. But, on the downside, by signing a transaction, W-OTS reveals parts of a private key for the specific address tokens are being spent from. 
 - With W-OTS, every time a signature is used to spend tokens from a particular address, any remaining tokens need to be moved onto a new address to prevent malicious actors from brute-forcing (trial-and-error guessing) the remaining parts of the private key for the address. That’s the main reason why this signature scheme is considered to be a “one-time signature”. 
-- So, after our Chrysalis update, we are now using the Ed25519, based on the EdDSA, scheme instead of W-OTS. The advantage is that the new scheme addresses all of the issues that W-OTS originally had. Ed25519 verifies both single-signature and batch verification (taking care of the left over/remaining tokens) very quickly, as well as faster key generation and smaller signatures which make it very secure. 
+- So, after our Chrysalis update, we are now using the Ed25519, based on the EdDSA, scheme instead of W-OTS. The advantage is that the new scheme addresses all of the issues that W-OTS originally had. Ed25519 verifies both single-signature and batch verification (taking care of the remaining tokens) very quickly, as well as faster key generation and smaller signatures which make it very secure. 
 
 ## What is Bundle Mining?
+
 -  If you have spent addresses, it means you accidentally received funds to an address that was already spent from, and these funds are not safe to send again due to W-OTS. 
 -  To secure your spent addresses during the migration, [Firefly](https://github.com/iotaledger/firefly) will try to find a new bundle that reveals the least amount of additional private key parts compared to previous signatures.
-- This process will take 10 minutes per spent address and upon completion you will be presented with a risk calculation (very high, high, medium, low, very low). We recommended that you repeat the process if it returns a bundle with medium risk or higher, particularly for significant sums of IOTA. You have the option to select which addresses you want to mine for, and again which you want to rerun the process for.
+- This process will take 10 minutes per spent address and upon completion, you will be presented with a risk calculation (very high, high, medium, low, very low). We recommended that you repeat the process if it returns a bundle with medium risk or higher, particularly for significant sums of IOTA. You have the option to select which addresses you want to mine for, and again which you want to rerun the process for.
```

Comment:
I disagree. This comma disconnects "upon completion" from the rest of the clause, so now it is not immediately clear to which clause it belongs to, even if technically proper.

Reply:
Difference in styles🤷; wont fight this comma but you know there will be more.